packages/build/devserver/index.js

```diff
@@ -112,7 +113,12 @@ function serveIndexHtml(req, res) {
   }
 
   function handleRequest() {
-    proxyServer.web(req, res, getTargetOptions());
+    proxyServer.web(req, res, getTargetOptions(), (err, req, res) => {
+      const msg = `error handling request: ${err.message}. Is the target running and accessible at ${target}?`;
+      console.log(msg);
```

Comment:
For other errors in `devserver` we use `console.error`, so we can use that function to be consistent (but the output looks the same).

Reply:
Awesome, thanks